Resources/Locale/en-US/seeds/seeds.ftl

```diff
@@ -117,3 +117,5 @@ seeds-cotton-name = cotton
 seeds-cotton-display-name = cotton plant
 seeds-pyrotton-name = pyrotton
 seeds-pyrotton-display-name = pyrotton plant
+seeds-coffee-name = Coffee
+seeds-coffee-display-name = Coffee Bush
```

Comment:
Consistent capitalization.  Also, consider moving to Resources/Locale/en-US/_NF/seeds/seeds.ftl to avoid merge conflicts from upstream changes.
```suggestion
seeds-coffee-name = coffee
seeds-coffee-display-name = coffee plant
```